HomeWork/Façade/Calculator.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Numerics;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Façade
+{
+    class Calculator
+    {
+        static readonly ArithmeticOperations _arithmeticOperations = new ArithmeticOperations();
```

Comment:
Please, always put a modificator of access even you use a default one. Such code becomes more readable..